openpower/package/openpower-pnor-p10/0001-P10-Increase-PAYLOAD-partition-size-to-1MB.patch

```diff
@@ -0,0 +1,29 @@
+From bc7eb1924e5de56c0bbc691ad26537fd5cd657b1 Mon Sep 17 00:00:00 2001
+From: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
+Date: Wed, 28 Jul 2021 15:20:00 +0530
+Subject: [PATCH] P10: Increase PAYLOAD partition size to 1MB
+
+After adding P10 support PAYLOAD (skiboot) size is crossing
+512K size. Hence increase PAYLOAD size to 1MB.
+
+Signed-off-by: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
+---
+ p10Layouts/defaultPnorLayout_64.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/p10Layouts/defaultPnorLayout_64.xml b/p10Layouts/defaultPnorLayout_64.xml
+index c2ebc14..9d75f08 100644
+--- a/p10Layouts/defaultPnorLayout_64.xml
++++ b/p10Layouts/defaultPnorLayout_64.xml
+@@ -173,7 +173,7 @@ Layout Description
+     <section>
+         <description>Payload (512KiB)</description>
```

Comment:
The Table of Contents will be automatically updated, so this change doesn't have any other dependencies. Just need to update the description with the new size.

Reply:
Thanks @anoo1 . We will push changes to pnor repository. 

-Vasant